releases.yaml

```diff
@@ -42,7 +42,6 @@ checksums:
   desktop-arm64+raspi:
     "22.04": "75d05ef58e527b36ffb7807eae23d6b63cc80adaf59fff95c39f4077647a6423 *ubuntu-22.04-preinstalled-desktop-arm64+raspi.img.xz"
     "21.10": "5187d507099f26bc4d8218085109af498fae5ff93b40c668f83bab5c7574d954 *ubuntu-21.10-preinstalled-desktop-arm64+raspi.img.xz"
-    "21.04": "d89ee327a00b98d7166b1a8cc95d17762aaacd3b4d0fc756c5b6b65df1708f48 *ubuntu-21.04-preinstalled-desktop-arm64+raspi.img.xz"
```

Comment:
Why has this one been removed?

Reply:
Hi Anthony, 21.04 is removed from the pages and it's in the EOL. So remove it.